mapactionpy_controller/xml_exporter.py

```diff
@@ -44,24 +44,24 @@ def setExportParameters(self, params):
         exportPropertiesDict["product-type"] = params["productType"]
         exportPropertiesDict["summary"] = params["summary"]
         # Fixed values
-        exportPropertiesDict["imagerydate"] = ""
-        exportPropertiesDict["papersize"] = "A3"
-        exportPropertiesDict["access"] = "MapAction"  # Until we work out how to get the values for this
-        exportPropertiesDict["accessnotes"] = ""
-        exportPropertiesDict["location"] = ""
-        exportPropertiesDict["qclevel"] = "Automatically generated"
-        exportPropertiesDict["qcname"] = ""
-        exportPropertiesDict["proj"] = ""
-        exportPropertiesDict["datasource"] = ""
-        exportPropertiesDict["kmlresolutiondpi"] = ""
-        exportPropertiesDict["paperxmax"] = ""
-        exportPropertiesDict["paperxmin"] = ""
-        exportPropertiesDict["paperymax"] = ""
-        exportPropertiesDict["paperymin"] = ""
-        exportPropertiesDict["createdate"] = None
-        exportPropertiesDict["createtime"] = None
-        exportPropertiesDict["scale"] = None
-        exportPropertiesDict["datum"] = None
+        exportPropertiesDict["imagerydate"] = params.get('imagerydate', "")
```

Comment:
There are a lot of lines here which just do "copy some value from params to exportPropertiesDict"
then there are some lines that do "copy some value from params to exportPropertiesDict or apply a default if it's missing". Finally there are some more interesting lines that apply some logic eg:
```
if (params["versionNumber"] == 1):
            exportPropertiesDict["status"] = "New"
        else:
            exportPropertiesDict["status"] = "Update"
```

Can I suggest separating out these three groups like so:
```
# statically define the default values
exportPropertiesDict = {
	'papersize': 'A3'
	'qclevel', 'Automatically generated'
}

# overwrite any default values from params
exportPropertiesDict.update(params)

# Now do interesting things
if (exportPropertiesDict["versionNumber"] == 1):
            exportPropertiesDict["status"] = "New"
        else:
            exportPropertiesDict["status"] = "Update"
```